desktop/ui/src/main/java/org/datacleaner/widgets/options/DataCloudOptionsPanel.java

```diff
@@ -160,11 +162,15 @@ private boolean updateDcConfiguration() {
         try {
             RemoteServersUtils.checkServerWithCredentials(DATACLOUD_URL, username, password);
         } catch (Exception ex) {
+            invalidCredentialsLabel.setForeground(new Color(170,10,10));
```

Comment:
How come this doesn't use the normal color scheme in WidgetUtils (the old apparently didn't either, so it may not be anything new)?